x/axelarcork/client/cli/query.go

```diff
@@ -0,0 +1,349 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/peggyjv/sommelier/v6/x/axelarcork/types"
+	"github.com/spf13/cobra"
+)
+
+// GetQueryCmd returns the cli query commands for this module
+func GetQueryCmd() *cobra.Command {
+	corkQueryCmd := &cobra.Command{
+		Use:                        "axelar-cork",
```

Comment:
The name was because there is inconsistency between legal names in go, proto and rust. In terms of readability, it is probably better not to mash the word. But I can remove the hyphen if you like

Reply:
I think the idiomatic way is to use the exact module name, e.g. that's the way cellarfees is now, I'm sure there are probably examples on other chains as well.

Reply:
Changed